ep/build-velox/src/build_velox.sh

```diff
@@ -65,6 +65,19 @@ for arg in "$@"; do
   esac
 done
 
+function apply_compilation_fixes {
+  current_dir=$1
+  velox_home=$2
+  sudo cp ${current_dir}/compilation.patch ${velox_home}/
+  sudo mkdir -p ${velox_home}/third_party/arrow_patches/
+  sudo cp ${current_dir}/helpers.patch ${velox_home}/third_party/arrow_patches/
+  cd ${velox_home}
+  git apply compilation.patch
+  if [ $? -ne 0 ]; then
+    echo "Failed to apply compilation fixes to Velox: $?."
```

Comment:
Please add `exit 1` to directly make the build exit. Suppose it is not acceptable to continue the build without the patch applied.

Reply:
Updated.